frontend/package-lock.json

```diff
@@ -0,0 +1,1769 @@
+{
```

Comment:
I guess you are using npm. Wanna try yarn or pnpm? [Why?](https://dev.to/ayoub3bidi/lets-settle-things-out-2-npm-vs-yarn-vs-pnpm-5e04#:~:text=For%20NPM%20It%20is%20still,PNPM%20is%20faster%20than%20Yarn.)